### PR TITLE
Add new Dutch policy report

### PR DESCRIPTION
**Pull request by ElenaFdR**
Can someone with stronger Dutch than me please sense check my Dutch translation?

-----
[View rendered activities/value-and-impact/policy-documents.md](https://github.com/publiccodenet/about/blob/Add-new-Dutch-policy-report/activities/value-and-impact/policy-documents.md)

### CODE REVIEW

File: activities/value-and-impact/policy-documents.md [hunk 1]
```diff
@@ -7,6 +7,7 @@ explains: where we are mentioned in public policy documents
 
 ## 2022
 
+* Netherlands Ministry of the Interior and Kingdom Relations: [Open source working: beyond the voluntary approach](https://www.rijksoverheid.nl/documenten/rapporten/2022/09/13/opensourcewerken-de-vrijblijvendheid-voorbij)
```
Comment (clausmullie) [inline activities/value-and-impact/policy-documents.md:10]:
```suggestion
* Ministry of the Interior and Kingdom Relations of  Netherlands: [Open source working: beyond the voluntary approach](https://www.rijksoverheid.nl/documenten/rapporten/2022/09/13/opensourcewerken-de-vrijblijvendheid-voorbij)
```

Comment (clausmullie) [inline activities/value-and-impact/policy-documents.md:10]:
```suggestion
* Netherlands Ministry of the Interior and Kingdom Relations: [Open source working: beyond the non-binding approach](https://www.rijksoverheid.nl/documenten/rapporten/2022/09/13/opensourcewerken-de-vrijblijvendheid-voorbij)
```

**Review by clausmullie — COMMENTED** [review]
Two suggestions

**Review by ericherman — DISMISSED** [review]

File: activities/value-and-impact/policy-documents.md [hunk 2]
```diff
@@ -7,6 +7,7 @@ explains: where we are mentioned in public policy documents
 
 ## 2022
 
+* Ministry of the Interior and Kingdom Relations of  Netherlands: [Open source working: beyond the voluntary approach](https://www.rijksoverheid.nl/documenten/rapporten/2022/09/13/opensourcewerken-de-vrijblijvendheid-voorbij)
```
Comment (ElenaFdR) [inline activities/value-and-impact/policy-documents.md:10]:
```suggestion
* Ministry of the Interior and Kingdom Relations of  Netherlands: [Open source working: beyond the non-binding approach](https://www.rijksoverheid.nl/documenten/rapporten/2022/09/13/opensourcewerken-de-vrijblijvendheid-voorbij)
```

Comment (ericherman) [inline activities/value-and-impact/policy-documents.md]: Is the word "the" missing here?

File: activities/value-and-impact/policy-documents.md [hunk 3]
```diff
@@ -7,6 +7,7 @@ explains: where we are mentioned in public policy documents
 
 ## 2022
 
+* Ministry of the Interior and Kingdom Relations of  Netherlands: [Open source working: beyond the non-binding approach](https://www.rijksoverheid.nl/documenten/rapporten/2022/09/13/opensourcewerken-de-vrijblijvendheid-voorbij)
```
Comment (ericherman) [inline activities/value-and-impact/policy-documents.md:10]: Should this read "the Netherlands"

**Review by ericherman — CHANGES_REQUESTED** [review]

File: activities/value-and-impact/policy-documents.md (hunk 3 above)
Comment (ElenaFdR) [reply activities/value-and-impact/policy-documents.md:10]: Google for some reason renders it without in its info sidebar, but I can't find any official or unofficial doc that also does that. I'll add it back.

Comment (ElenaFdR) [inline activities/value-and-impact/policy-documents.md:10]:
```suggestion
* Ministry of the Interior and Kingdom Relations of the Netherlands: [Open source working: beyond the non-binding approach](https://www.rijksoverheid.nl/documenten/rapporten/2022/09/13/opensourcewerken-de-vrijblijvendheid-voorbij)
```

**Review by ericherman — APPROVED** [review]

**Pull request merged**